cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -34,48 +38,113 @@ var (
 	cssTaskList   = []models.Task{}
 	exceptionList = []Exception{}
 
-	taskDefnARN = ""
+	taskDefnARN         = ""
+	roleName            = "E2ETestRole"
+	instanceProfileName = "E2ETestInstance"
+	policyARN           = "arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role"
 )
 
+const errorCode = 1
```

Comment:
I don't see where this variable `errorCode` is used anywhere. I assume it's supposed to be used in the `os.Exit` calls?